product_offer/models/product_template.py

```diff
@@ -70,6 +70,19 @@ class ProductTemplate(models.Model):
         readonly=True,
     )
 
+    net_price_cny = fields.Float(
+        string='Sale RMB',
+        compute='_get_net_price_cny',
+        digits=dp.get_precision('Product Price'),
+    )
+
+    @api.multi
+    def _get_net_price_cny(self):
+        cny_rec = self.env['res.currency'].search([('name', '=', 'CNY')])[0]
+        if cny_rec:
+            for pt in self:
+                pt.net_price_cny = pt.net_price * cny_rec.rate_silent
+
```

Comment:
Two issues:

1) We already have the field here: https://github.com/rfhk/awo-custom/blob/ef65ffdd4c951d9c1bf6661e1a37233d8612b365/product_listprice_list_view/models/product_template.py#L41-L45, so it is redundant to repeat the definition

2) If anything, I guess you probably want to show the price without decimal values.  We prepared `net_price_integer` field for this reason.

Since `product_listprice_list_view` module depends on `product_offer`, you might want to move the field definition plus associated method (which I guess is kind of what you are doing here, but you need to remove these parts from `product_listprice_list_view` at the same time.